modules/mobile/app/controllers/mobile/v0/appointments_controller.rb

```diff
@@ -48,7 +48,7 @@ def cancel
       def create
         new_appointment = appointments_helper.create_new_appointment(params)
         serializer = VAOS::V2::VAOSSerializer.new
-        serialized = serializer.serialize(new_appointment, 'appointmentRequest')
+        serialized = serializer.serialize(new_appointment, 'appointment')
```

Comment:
I changed this to match what the web app is doing. All it does is set the type in the serializer. I could make it dependent upon which status is received, but I don't see that it helps anything since this will already be clear to the FE from the status.